### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by sserkov84**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
```
Comment (NazarSheremet) [inline app/main.py:20]: The solution is correct, but it's better to use the `count()` function as it will simplify the code.

**Review by NazarSheremet — CHANGES_REQUESTED** [review]
Need to optimіze solution

**Comment by sserkov84** [comment]
Optimized as needed

**Review by YuriiSmolii — APPROVED** [review]
Well done.

**Pull request closed without merging**